opencti-platform/opencti-front/src/components/ViewSwitchingButtons.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { FunctionComponent } from 'react';
+import ToggleButton from '@mui/material/ToggleButton';
+import Tooltip from '@mui/material/Tooltip';
+import { LibraryBooksOutlined, ViewModuleOutlined } from '@mui/icons-material';
+import { FormatListGroup, Group, RelationManyToMany, VectorPolygon } from 'mdi-material-ui';
+import { ToggleButtonGroup } from '@mui/material';
+import { ListViewIcon, SublistViewIcon } from 'filigran-icon';
+import { makeStyles } from '@mui/styles';
+import FiligranIcon from '@components/common/FiligranIcon';
+import { useFormatter } from './i18n';
+
+const useStyles = makeStyles({
+  EntityIcon: {
+    height: 16,
+    width: 16,
+  },
+  subEntityButton: {
+    height: 36,
+  },
+});
```

Comment:
Do not use `makeStyles` which is deprecated. Put styles directly in the JSX using `style={{...}}`